### ISSUE TITLE
Error when disk name is too long

### ISSUE THREAD

**Issue by Guimove** [CentralReport/CentralReport#78]
I found this issue working on CentOS detection ( #58 )

When a systemfile is too long, the function get_disks fails with this error : 

``` Python
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/local/lib/centralreport/cr/threads.py", line 92, in run
    Checks.last_check_disk = self.MyCollector.get_disks()
  File "/usr/local/lib/centralreport/cr/collectors.py", line 437, in get_disks
    disk_free = int(line_split[3]) * 1024
IndexError: list index out of range
```

When you see the output of the 'df' command, you see : 

![capture_d_cran_04_06_13_22_11](https://f.cloud.github.com/assets/1172067/608025/e8eba4f0-cd52-11e2-8e72-11e5ea436da7.png)

The first line is truncated by the lenght of '/dev/mapper/vg_centcro-lv_root'

So when you parse this output with  the function get_disks, the list line_split has only one raw : 

``` Bash
[/dev/mapper/vg_centcro-lv_root]
```

But with a normal output (like in debian), you get 

``` Bash
[/dev/sda1, 19751804, 1016984, 17731456, 6%, /]
```

IMO, We have to find an other way to parse this output when the name is to much long.

**Comment by miniche** [comment]
I've read the man page for the df command, and I think that two arguments may be very useful for us:
- **-k** to force 1K unit (in fact, the default value is 1K, but can be 512 octets if the global variable "POSIXLY_CORRECT" is defined on the host)
- **-P** to use the POSIX output format, with columns.

The **df -kP** command on my CentOS returns:
![capture decran 2013-06-06 a 00 30 45](https://f.cloud.github.com/assets/1638158/615126/b0c4473e-ce2f-11e2-9cf4-c9cb7ce578b7.png)

And on Debian:
![capture decran 2013-06-06 a 00 30 56](https://f.cloud.github.com/assets/1638158/615130/c0968fbe-ce2f-11e2-8e95-84c881fb21ad.png)

Man page in english: http://unixhelp.ed.ac.uk/CGI/man-cgi?df
Man page in french: http://pwet.fr/man/linux/commandes/df

**Comment by Guimove** [comment]
That's working thanks my guru

**Comment by miniche** [comment]
Fixed with 59cd274444a21b47e191001998dbcdc779bb338e
